src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php

```diff
@@ -203,8 +206,16 @@ public function testTwigLoaderPaths($format)
             array('namespaced_path3', 'namespace3'),
             array(__DIR__.'/Fixtures/Resources/TwigBundle/views', 'Twig'),
             array(realpath(__DIR__.'/../..').'/Resources/views', 'Twig'),
+            array(__DIR__.'/Fixtures/Bundle/ChildTwigBundle/Resources/views', 'ChildTwig'),
+            array(__DIR__.'/Fixtures/Bundle/ChildChildTwigBundle/Resources/views', 'ChildChildTwig'),
             array(__DIR__.'/Fixtures/Resources/views'),
         ), $paths);
+
+        $this->assertEquals(array(
+            array(__DIR__.'/Fixtures/Bundle/ChildTwigBundle/Resources/views', 'Twig'),
+            array(__DIR__.'/Fixtures/Bundle/ChildChildTwigBundle/Resources/views', 'ChildTwig'),
+            array(__DIR__.'/Fixtures/Bundle/ChildChildTwigBundle/Resources/views', 'Twig'),
+        ), $prependPaths);
```

Comment:
the issue is that the way the test is written does not allow to quickly validate what the final order will be when looking at it. This is why I think the test should not be written this way. We don't want to assert 2 separate lists, as we care about a single list (the one being used at runtime).

thus, the test is even confusing when looking at it, because the list of ``prependPath`` here is in reverse order compared to the order of paths in the loader (as prepending adds at the beginning while you append in ``$prependPaths``), making it even harder to validate/understand the assertion.